### PR TITLE
Fix library position and mobile regression.

### PR DESCRIPTION
**Pull request by jasmussen**
This PR fixes two things.

1. When invoking the block library from the plus on the side, it aligns it correctly

2. On mobile breakpoints, the "recently used" blocks were shown on the right even though they should be hidden.

After:

screen shot 2018-08-21 at 09 28 01

screen shot 2018-08-21 at 09 34 55

Before:

screen shot 2018-08-21 at 09 29 02

### CODE REVIEW

File: packages/editor/src/components/inserter-with-shortcuts/style.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .editor-inserter-with-shortcuts {
-	display: flex;
+	//display: flex;
```
Comment (gziolo) [inline packages/editor/src/components/inserter-with-shortcuts/style.scss]: Is that intended?

Comment (jasmussen) [reply packages/editor/src/components/inserter-with-shortcuts/style.scss]: Raaargh no, sorry. One sec fixing.

Comment (jasmussen) [reply packages/editor/src/components/inserter-with-shortcuts/style.scss]: Sorry about that. Fixed not only this typo, but another one. 

screen shot 2018-08-21 at 10 00 35

screen shot 2018-08-21 at 10 01 11

**Comment by noisysocks** [comment]
I'm noticing that if I click into the default block appender and then click away from it, the block shortcuts cover the placeholder text:

screen shot 2018-08-22 at 11 10 52

**Review by noisysocks — CHANGES_REQUESTED** [review]
I'm noticing that if I click into the default block appender and then click away from it, the block shortcuts cover the placeholder text:

screen shot 2018-08-22 at 11 10 52

**Comment by jasmussen** [comment]
Solid review, thank you. I believe I've addressed all points now:

screen shot 2018-08-22 at 09 59 07

screen shot 2018-08-22 at 09 59 13

screen shot 2018-08-22 at 09 59 19

screen shot 2018-08-22 at 09 59 29

screen shot 2018-08-22 at 10 02 51

**Review by noisysocks — APPROVED** [review]
It's beautiful 😍

**Comment by jasmussen** [comment]
😅

**Pull request merged**